seed/api/v2_1/views.py

```diff
@@ -169,6 +170,23 @@ def building_sync(self, request, pk):
               required: true
               paramType: query
         """
```

Comment:
Not sure where any potential code changes would end up, but relevant to this endpoint...

Did some monkey-testing with exporting and saw some weird stuff happening. See the image below (left: exported xml, top-right: preset chosen during export and value of **Address Line 2**, bottom-right: preset setting with xpath for **Address Line 2**):
![Screen Shot 2020-05-13 at 2 40 11 PM](https://user-images.githubusercontent.com/30608004/81863395-5a092280-9528-11ea-9a16-f27ba9cf9199.png)

I would've expected `StreetAddress` xml node to be empty. But not only was it populated, it was populated with a previous Address Line **1** value (instead of an Address Line 2 value).

Reply:
Per our conversation, this is expected. The process for building the BuildingSync XML export takes the previous import (with all it's values) and updates the values of that file using presets and the latest -State. 

@macintoshpie, thinking about this more, this pattern (of starting with the original xml upload and building off that) is great for keeping nodes/values that we never even looked at during import. But should "empty" or None values replace the original values for nodes that we did read from on import? In this case, would it be appropriate to make the StreetAddress node empty? 

Definitely just thinking out loud - I'm not sure if this makes sense in the larger BuildingSync context.

Reply:
From an implementation standpoint that might complicate things. E.g. if we reomve auc:StreetAddress, should we prune back the xml tree and how far?

I think @nllong would have a better idea as to whether it would make sense to remove elements from the tree

Reply:
Sounds like BuildingSync doesn't have the notion of a node that has an empty value?

Reply:
Created issue #2224 to track this